test/e2e/scenarios/retina/retina_scenarios_test.go

```diff
@@ -71,12 +77,29 @@ func TestE2ERetinaMetrics(t *testing.T) {
 		Namespace:   "kube-system",
 		ReleaseName: "retina",
 		ChartPath:   "../../../../deploy/manifests/controller/helm/retina/",
-	}, nil)
+	}, &types.StepOptions{
+		SkipSavingParamatersToJob: true,
+	})
 
 	job.AddScenario(drop.ValidateDropMetric())
 
 	// todo: handle multiple scenarios back to back
 	job.AddScenario(tcp.ValidateTCPMetrics())
 
+	// check advanced metrics
+	job.AddScenario(dns.ValidateDNSMetric())
+
+	// enable advanced metrics
```

Comment:
@matmerr 
What do you think about using table driven testing for different Retina modes? Don't think a single Test function is sustainable for all scenarios.

Reply:
Yep, tables make it more auditable for sure, we could make it something like?

    type DNSScenario struct {
        input string
    }

    tests := []DNSScenario{
        {input: "abc"},
        {input: "def"}
    }

    for _, tc := range tests {
        job.AddScenario(dns.ValidateDNSMetric(tc))
    }

Reply:
I think we should break this up into 

```bash
type Template struct {
    name string
    steps []steps
}

// Define in separate files
basic := Template{"basic", []steps{...})
adv := Template{"advance", []steps{...}}

// Small framework to start tests.
func TestE2e(t *testing.T) {
    testCases := []Template{basic, adv}
    for testCase in range(testCases):
        t.Run(...)
    }
}
```